st2common/st2common/policies/meta/concurrency.yaml

```diff
@@ -6,5 +6,13 @@ resource_type: action
 module: st2actions.policies.concurrency
 parameters:
     threshold:
+        description: Concurrency threshold.
         type: integer
         required: true
+    action:
```

Comment:
/cc @m4dcoder @enykeev what others think about the attribute name?

I can't come up with anything better than `action`.